### PR TITLE
Ag changes

### PR DESCRIPTION
**Pull request by akeller**
* Updated banner with triangle asset
* Updated button color & hover

### CODE REVIEW

**Comment by menski** [comment]
FYI the change "glossary updates for process terminology" (9a54a84) was already merge with PR #190

**Comment by akeller** [comment]
> FYI the change "glossary updates for process terminology" ([9a54a84](https://github.com/camunda-cloud/camunda-cloud-documentation/commit/9a54a84c4e775bffd8489ebeaee669a111df8c68)) was already merge with PR #190

I removed it with a rebase. I also modified this PR to merge into `master` so we can push changes tomorrow.

**Review by menski — APPROVED** [review]
lgtm

File: docs/product-manuals/zeebe/open-source/deprecated-features.md [hunk 1]
```diff
@@ -32,3 +32,7 @@ In terms of specifying values, there were two minor changes:
 
 - Memory sizes are now specified like this: `512MB` (old way: `512M`)
 - Durations, e.g. timeouts, can now also be given in ISO-8601 Durations format. However you can still use the established way and specify a timeout of `30s`
+
+## Removed in 1.0
```
Comment (pihme) [inline docs/product-manuals/zeebe/open-source/deprecated-features.md:36]: Being a bit nit-picky here: Since 1.0 is not released yet this currently describes the future. So we could either remove, or rephrase it to make it clearer that it will be removed.

File: docs/product-manuals/clients/cli-client/index.md [hunk 2]
```diff
@@ -54,7 +54,7 @@ Available Commands:
   cancel      Cancel resource
   complete    Complete a resource
   create      Create resources
-  deploy      Creates new workflow defined by provided BPMN or YAML file as workflowPath
+  deploy      Creates new workflow defined by provided BPMN file as workflowPath
```
Comment (pihme) [inline docs/product-manuals/clients/cli-client/index.md:57]: As far as I know there is a deliberate renaming under way which replaces all instances of `workflow` with `process`. I am not sure whether this applies only to the code or also to the documentation.

See e.g. https://github.com/zeebe-io/zeebe/pull/6455

@menski can say more about this and give some context

File: docs/reference/grpc.md [hunk 3]
```diff
@@ -287,14 +287,16 @@ message WorkflowRequestObject {
     // using the file extension of the name field
     FILE = 0;
     BPMN = 1; // extension 'bpmn'
-    YAML = 2; // extension 'yaml'
+    YAML = 2 [deprecated = true]; // extension 'yaml'; removed as of release 1.0
```
Comment (pihme) [inline docs/reference/grpc.md:290]: again, not released yet

File: docs/reference/grpc.md [hunk 4]
```diff
@@ -287,14 +287,16 @@ message WorkflowRequestObject {
...
   }
 
   // the resource basename, e.g. myProcess.bpmn
   string name = 1;
   // the resource type; if set to BPMN or YAML then the file extension
   // is ignored
-  ResourceType type = 2;
+  // As of release 1.0, YAML support was removed and BPMN is the only supported resource type.
```
Comment (pihme) [inline docs/reference/grpc.md:297]: again, not released yet

File: versioned_docs/version-0.26/guides/getting-started/connect-to-your-cluster.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+id: connect-to-your-cluster
+title: Connect to your cluster
+---
+
+## Installation
+
+Install the appropriate package.
+
+```bash
+npm i -g zbctl
```
Comment (pihme) [inline versioned_docs/version-0.26/guides/getting-started/connect-to-your-cluster.md:11]: We had a discussion internally whether this is the best installation source. I think the NPM package is maintained by Josh and not up to date all the time. On the plus side it is a very easy installation.
Our internal discussion didn't yield any clear preference, so it might be up for product management to decide whether they want to elevate the NPM package to officially supported or reduce its usage,

**Review by pihme — APPROVED** [review]
Some comments to consider. I didn't look at all the files, though. was a bit overwhelming.

**Comment by akeller** [comment]
@pihme thanks for your comments. When I opened the PR I noticed these came over, possibly because I was originally merging into 'release_0_26'. Since these are not my changes and I'd like to merge this PR today I can remove the ones that are not mine and referenced in the description.

**Review by menski — APPROVED** [review]
:shipit:
